lib/arclight/hash_absolute_xpath.rb

```diff
@@ -24,7 +24,7 @@ def initialize(node)
     end
 
     def to_hexdigest
-      self.class.hash_algorithm.hexdigest(absolute_xpath)
+      self.class.hash_algorithm.hexdigest(absolute_xpath).prepend('al_')
```

Comment:
Could we re-verify this again? I was able to get this working without prepending `al`. Could we chat quickly about this?

Reply:
Thanks for pairing up @seanaery on this. I was able to see t he issue in the "Contents" collapse and expand and was confused.